### PR TITLE
Changes of navijgation bar to develop

### PR DESCRIPTION
**Pull request by Lingtao-Cai**
1. changes the color of items in the navigation bar
2. fix bugs in multiChat when there is an empty chat
3. fix the size of feed in fragment
4. make all fragment to new when it is replaced

### CODE REVIEW

**Review by HellinSun — APPROVED** [review]
Nice work Lingtao!

**Review by arfi2212 — APPROVED** [review]

File: app/src/main/java/com/example/ar_reshare/SwipeActivity.java [hunk 1]
```diff
@@ -36,12 +36,7 @@ public class SwipeActivity extends AppCompatActivity implements NavigationBarVie
     private final int TAP_OFFSET = 10;
     private boolean touchedDown = false;
     private boolean moved = false;
-
-    Fragment chatListActivity = new ChatListActivity();
-    Fragment mapsActivity = new MapsActivity();
-    Fragment profileActivity = new ProfileActivity();
-    Fragment arActivity = new ARActivity();
-    Fragment feedActivity = new FeedActivity();
+    private boolean isArSupported=false;
```
Comment (artur-varosyan) [inline app/src/main/java/com/example/ar_reshare/SwipeActivity.java:39]: I think this should be static Lingtao?

**Review by artur-varosyan — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/ar_reshare/SwipeActivity.java (hunk 1 above)
Comment (Lingtao-Cai) [reply app/src/main/java/com/example/ar_reshare/SwipeActivity.java:39]: done!

Comment (artur-varosyan) [reply app/src/main/java/com/example/ar_reshare/SwipeActivity.java]: Thanks!

**Review by artur-varosyan — APPROVED** [review]
Thanks Lingtao!

**Pull request merged**